Formula/gcc-arm-embedded.rb

```diff
@@ -0,0 +1,43 @@
+class GccArmEmbedded < Formula
+  desc "GNU Embedded Toolchain for Arm Cortex-M/R processors"
+  homepage "https://developer.arm.com/open-source/gnu-toolchain/gnu-rm"
+  url "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/RC2.1/src%202.0/gcc-arm-none-eabi-9-2019-q4-major-src.tar.bz2"
+  version "9-2019-q4-major"
+  sha256 "f162a655f222319f75862d7aba9ff8a4a86f752392e4f3c5d9ef2ee8bc13be58"
+
+  # build-prerequisites fails without this, when ./configure is run from zlib it raises an error,
+  # the zlib log shows the problem is: "The build tool has reset ENV; --env=std required."
+  env :std
+
+  depends_on "gnu-tar" => :build
+  depends_on :xcode => ["10.0", :build]
+
+  def install
+    inreplace "build-common.sh", "JOBS=1", "JOBS=`sysctl -n hw.ncpu`"
```

Comment:
Let's not do that, if it insists on one job, maybe there is a reason. Report it as a bug upstream.

Reply:
Okay. I will take it out. Just for reference I have asked the maintainer why and the response I received can be found [here](https://answers.launchpad.net/gcc-arm-embedded/+question/685668). Seeing how this build failed on the test bot for only High Sierra and Mojave I think removing this line of code is necessary.

Can I ask if all of the different test bot versions have the APFS file system?